hs_model_program/migrations/0005_auto_20150428_2243.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import datetime
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hs_model_program', '0004_auto_20150422_2052'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='mpmetadata',
+            name='date_released',
+            field=models.DateTimeField(default=datetime.datetime(2015, 4, 28, 22, 43, 24, 183917), help_text=b'The date of the software release (m/d/Y H:M)', null=True, verbose_name=b'Release Date', blank=True),
```

Comment:
Hi Calvin, I didn't intentionally make these migration changes. How can I address it so that we can have this branch merged into develop asap?